src/index.ts

```diff
@@ -18,6 +18,7 @@ export {
 } from './commons/new';
 export * as Incubator from './incubator';
 export * as Hooks from './hooks';
+export * as Modifiers from './commons/modifiers';
```

Comment:
I think this is incorrect. 
This will export all Modifiers functions separately, while we want to export it as a whole class.

Reply:
You can see the usage in the private lib's PR, it is as follows:
```
import {Modifiers} from 'react-native-ui-lib';
{...Modifiers.extractAccessibilityProps(this.props)}
```
Is that not what you meant?

Reply:
yes.. it's actually correct. 
Not sure why it look weird before :/
merging..